### PR TITLE
chore: release 0.38.0

### PR DESCRIPTION
**Pull request by ThibaultFy**

### CODE REVIEW

File: docs/source/additional/release.rst [hunk 1]
```diff
@@ -31,6 +31,23 @@ This is an overview of the main changes, please have a look at the changelog of
...
 - `orchestrator changelog <https://github.com/Substra/orchestrator/blob/main/CHANGELOG.md>`__
 
+Substra 0.38.0 --- 2024-06-13
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**All**
+
+- The release 0.38.0 focuses on improving the security and limit the rights of the differents Substra pods when deployed on a cluster. The following changes have been made:
+
+  - Ressources requests and limits are now set as default for the backend and frontend.
```
Comment (thbcmlowk) [inline docs/source/additional/release.rst:41]: Did we not set default values for the orchestrator?

**Review by thbcmlowk — APPROVED** [review]

File: docs/source/additional/release.rst (hunk 1 above)
Comment (ThibaultFy) [reply docs/source/additional/release.rst:41]: It was already all set for the orc if I'm not mistaken

Comment (thbcmlowk) [reply docs/source/additional/release.rst:41]: It seems like it was introduced in chart version [8.5.0](https://github.com/Substra/orchestrator/blob/main/charts/orchestrator/CHANGELOG.md#850---2024-04-05)

Substra 0.37.0 targets chart version [8.4.0](https://github.com/Substra/substra-documentation/blob/main/docs/source/additional/releases.yaml#L32)

Comment (thbcmlowk) [reply docs/source/additional/release.rst:41]: Also I think I would reword a bit:
```suggestion
  - CPU and memory requests and limits are now provided by default for all orchestrator, backend and frontend pods.
```

Comment (ThibaultFy) [reply docs/source/additional/release.rst:41]: Ok nice catch thanks !!

File: docs/source/additional/release.rst [hunk 2]
```diff
@@ -31,6 +31,23 @@ This is an overview of the main changes, please have a look at the changelog of
...
+  - Network policies have been added to limit the communication between the different pods.
+  - Update the different base docker images of the Susbtra components and the the user's ones to limit their vulnerabilities.
+
+- The profiling of functions have been reworked to take into accound the ``builiding`` step and the ``download_function`` step.
+
+**Backend**
```
Comment (guilhem-barthes) [inline docs/source/additional/release.rst:47]: This is not only backend, but also orchestrator (or the title sounds confusing to me)

File: docs/source/additional/release.rst [hunk 3]
```diff
@@ -31,6 +31,23 @@ This is an overview of the main changes, please have a look at the changelog of
...
+
+**All**
+
+- The release 0.38.0 focuses on improving the security and limit the rights of the differents Substra pods when deployed on a cluster. The following changes have been made:
+
+  - Ressources requests and limits are now set as default for the backend and frontend.
+  - Network policies have been added to limit the communication between the different pods.
+  - Update the different base docker images of the Susbtra components and the the user's ones to limit their vulnerabilities.
+
+- The profiling of functions have been reworked to take into accound the ``builiding`` step and the ``download_function`` step.
```
Comment (guilhem-barthes) [inline docs/source/additional/release.rst:45]:
```suggestion
- The profiling of functions have been reworked to take into accound the ``building`` step and the ``download_function`` step.
```

File: docs/source/additional/release.rst (hunk 2 above)
Comment (ThibaultFy) [reply docs/source/additional/release.rst]: ah my bad I thought it was only on backend, changed it

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+## [0.38.0](https://github.com/Substra/substra-documentation/releases/tag/0.38.0) - 2024-06-03
+
+### Added
+
+- Documentation on how to use the Harbor profile on substra-backend ([#416](https://github.com/Substra/substra-documentation/pull/416))
```
Comment (thbcmlowk) [inline CHANGELOG.md:14]: I am concerned there is not more added in the documentation in the light of the release notes. Did we not agree to document the NetworkPolicy part for instance? (cc @guilhem-barthes)

Comment (ThibaultFy) [reply CHANGELOG.md:14]: I added an entry about PSS that was not included in the changelog FYI.

Comment (thbcmlowk) [reply CHANGELOG.md:14]: Thank you. This should not block the release anyway, but I think we need to create follow-up issues.

**Pull request merged**